pkg/pss/pss.go

```diff
@@ -108,6 +109,9 @@ func (p *pss) Send(ctx context.Context, topic Topic, payload []byte, stamper pos
 
 	// push the chunk using push sync so that it reaches it destination in network
 	if _, err = p.pusher.PushChunkToClosest(ctx, tc); err != nil {
+		if errors.Is(err, topology.ErrWantSelf) {
```

Comment:
Will this be also replicated?

I think for pss to work, we will need to unwrap the trojan chunk. The handler in the pushsync does the unwrap. So I guess if this is replicated, it should be unwrapped.